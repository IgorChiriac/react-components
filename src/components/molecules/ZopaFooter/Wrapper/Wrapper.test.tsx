import { axe } from 'jest-axe';
import React from 'react';
import { render } from 'react-testing-library';
import Wrapper from './Wrapper';

describe('<Wrapper />', () => {
  it('renders the component', async () => {
    const { container } = render(<Wrapper />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
