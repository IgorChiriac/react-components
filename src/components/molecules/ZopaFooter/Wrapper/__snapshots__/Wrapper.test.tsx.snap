// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Wrapper /> renders the component 1`] = `
.c0 {
  padding: 18px 0;
}

<div
  class="c0"
/>
`;
